bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/Util.kt

```diff
@@ -16,9 +16,49 @@
 
 package org.modelix.model.sync.bulk
 
+import mu.KLogger
+import org.modelix.model.api.BuiltinLanguages
+import org.modelix.model.api.SimpleConcept
 import org.modelix.model.data.ModelData
 import org.modelix.model.data.NodeData
 
 fun mergeModelData(vararg models: ModelData): ModelData {
     return ModelData(root = NodeData(children = models.map { it.root }))
 }
+
+internal fun logImportSize(nodeData: NodeData, logger: KLogger) {
+    logger.debug { "Number of modules: ${countMpsModules(nodeData)}" }
+    logger.debug { "Number of models: ${countMpsModels(nodeData)}" }
+    logger.debug { "Number of concepts: ${countConcepts(nodeData)}" }
+    logger.debug { "Number of properties: ${countProperties(nodeData)}" }
+    logger.debug { "Number of references: ${countReferences(nodeData)}" }
+}
+
+private fun countProperties(data: NodeData): Int =
+    data.properties.size + data.children.sumOf { countProperties(it) }
+
+private fun countReferences(data: NodeData): Int =
+    data.references.size + data.children.sumOf { countReferences(it) }
+
+private fun countConcepts(data: NodeData): Int {
+    val set = mutableSetOf<String>()
+    countConceptsRec(data, set)
+    return set.size
+}
+
+private fun countConceptsRec(data: NodeData, set: MutableSet<String>) {
```

Comment:
```suggestion
    collectConcepts(data, set)
    return set.size
}

private fun collectConcepts(data: NodeData, set: MutableSet<String>) {
```

That function actually doesn't count

Reply:
What about this one?

Reply:
This function doesn't exist anymore. The concepts are now also collect in the new `measureImportSize()` function.

Reply:
Weird, the thread was still open when I browser through the individual commits.